app/cdap/components/ConnectionList/Components/ConnectionTabs/index.tsx

```diff
@@ -52,26 +52,34 @@ const ConnectionTab = styled(Tab)({
   },
 });
 
-export default function ConnectionsTabs({
+export default function({
   tabsData,
   handleChange,
   value,
   index,
   connectionId,
   setIsErrorOnNoWorkSpace,
-  ...props
-}) {
+  toggleLoader,
+}: IConnectionTabsProps) {
   const classes = useStyles();
 
-  const [connectionIdProp, setConnectionId] = useState(connectionId);
+  const [connectionIdProp, setConnectionId] = useState<string>(connectionId);
 
   useEffect(() => {
     setConnectionId(connectionId);
   }, []);
 
+  const handleConnectionTabClick = (connectorType: IConnectorTabType, index: number) => {
+    if (index > 1 && !connectorType.canBrowse) {
+      return;
```

Comment:
why we need if else statements when we have only one fulfilling condition
plz update

Reply:
check this plz

Reply:
Done